egs2/ljspeech/tts1/local/data.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=-1
+stop_stage=2
+
+log "$0 $*"
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+db_root=${LJSPEECH}
+
```

Comment:
Can add checking the value of LJSPEECH?
(I also forgot doing it in jsut)

https://github.com/espnet/espnet/blob/c997249cf96b803bb7293d9fbda6905a56ed9f93/egs2/commonvoice/asr1/local/data.sh#L25-L28

Reply:
Sure!

Reply:
Since we will download the corpus if not exists, `-z` is better?

Reply:
Oh, yes, you are right.